apps/frontend/src/pages/projects/components/project-create-form/project-create-form.tsx

```diff
@@ -45,6 +55,17 @@ const ProjectCreateForm = ({ onSubmit }: Properties): JSX.Element => {
 				name="description"
 				rowsCount={4}
 			/>
+			{charCount <= ProjectValidationRule.DESCRIPTION_MAXIMUM_LENGTH && (
+				<span className={styles["char-counter"]}>
+					{charCount}/{ProjectValidationRule.DESCRIPTION_MAXIMUM_LENGTH}
+				</span>
+			)}
+			{charCount > ProjectValidationRule.DESCRIPTION_MAXIMUM_LENGTH &&
+				!isSubmitted && (
+					<span className={styles["input-error"]}>
+						{ProjectValidationMessage.DESCRIPTION_TOO_LONG}
+					</span>
+				)}
```

Comment:
we should already have an error under the input if we exceed the description length by the project-create validation 
```suggestion
```

Reply:
There is a problem with this message appearing only when the submit button is pressed (validation starts only after the button is pressed). I have an idea how to avoid doing this check. We can change the color of the counter to red when the number of entered characters exceeds 1000. And after pressing the submit button, the user will see the error text. How is this solution?

Reply:
let's try to use [trigger](https://react-hook-form.com/docs/useform/trigger) from react-hook-form, to trigger validation on this field on change